src/pages/workspaces/workspace/analysis/modals/CloudEnvironmentModal.ts

```diff
@@ -127,31 +160,31 @@ export const CloudEnvironmentModal = ({
   const renderDefaultPage = () =>
     div(
       { style: { display: 'flex', flexDirection: 'column', flex: 1 } },
-      _.map((tool) => renderToolButtons(tool.label, cloudProvider))(
-        filterForTool ? [tools[filterForTool]] : getToolsToDisplayForCloudProvider(cloudProvider)
-      )
+      filterForTool
+        ? [renderToolButtons(tools[filterForTool].label, cloudProvider)]
+        : getToolsToDisplayForCloudProvider(cloudProvider).map((tool) => renderToolButtons(tool.label, cloudProvider))
     );
 
   const toolPanelStyles = {
     backgroundColor: 'white',
     margin: '0 1.5rem 1rem 1.5rem',
     padding: '0 1rem 1rem 1rem',
     display: 'flex',
-    flexDirection: 'column',
+    flexDirection: 'column' as const,
```

Comment:
it's because the ts "enums" react css uses requires the strings to be constant. To be honest, this was an autofix suggestion that made the error go away so I accepted it. If there's a better way to do this, please lmk

Reply:
Did a little digging. Seems to be an issue with the CSSType library.